### ISSUE TITLE
animation out problem

### ISSUE THREAD

**Issue by luiguild** [flauc/angular2-notifications#275]
Anyone know what it's happen with my notifications?

![gif](https://media.giphy.com/media/3o6fIYFO4PAn25eCRO/giphy.gif)

When the notification close, the animation out it's running twice...

**Comment by flauc** [comment]
Hi @luiguild 

Is that firefox?

**Comment by luiguild** [comment]
@flauc yes! but in Chrome I have the same behavior... I guess that it's the version... I changed to "@0.7.0" and it's works fine...

**Comment by flauc** [comment]
Interesting, will try to fix this asap, thank you for reporting it 👍

**Comment by quantumkoen** [comment]
I have the same in firefox (57.0.3 (64-bit)), but not in chrome (Version 63.0.3239.108 (Official Build) (64-bit))

**Comment by olingerc** [comment]
I can confirm this for firefox (57.0.4 (64-bit)) but working correctly in Chrome (Version 63.0.3239.108 (Official Build) (64-bit))
Additional info: Happens only if the timeout runs out, not on click to dismiss.

**Comment by olingerc** [comment]
Did you have a look at this?
(Sorry for the shameless bump. I tried finding a solution but did not find any)

**Comment by djoufouf** [comment]
same issue someone had found the solution ?
